src/utils/index.ts

```diff
@@ -1,13 +1,12 @@
-export { default as emitProgress } from './emitProgress';
-export { default as emitJobCompletion } from './emitJobCompletion';
-export { default as emitZipProgress } from './emitZipProgress';
-export { default as emitZipCompletion } from './emitZipCompletion';
-export { default as getOrigins } from './getOrigins';
-export { default as getMediaSrcFromRS } from './getMediaSrcFromRS';
-export { default as getTextFromRS } from './getTextFromRS';
-export { default as toBuff } from './toBuff';
-export { default as blobCache } from './blobCache';
-export { default as isViewableText } from './isViewableText';
-export { default as intoStream } from './intoStream';
-export { default as isNumber } from './isNumber';
-export { default as throwOutside } from './throwOutside';
+export { default as emitJobProgress } from './emit-job-progress';
+export { default as emitJobCompletion } from './emit-job-completion';
+export { default as emitZipProgress } from './emit-zip-progress';
+export { default as emitZipCompletion } from './emit-zip-completion';
+export { default as getOrigins } from './get-origins';
+export { default as blobCache } from './blob-cache';
+export { default as isViewableText } from './is-viewable-text';
+export { default as isNumber } from './is-number';
+export { default as throwOutside } from './throw-outside';
+export { default as parseBase64OrUint8Array } from './parse-base64-or-uint8array';
+export { default as emitError } from './emit-error';
+export { default as onPenumbraEvent } from './forward-events';
```

Comment:
Optional, but perhaps we could switch to named exports in these files to remove the need to do this re-exporting?